buildroot/package/con2fbmap/con2fbmap.mk

```diff
@@ -0,0 +1,25 @@
+#############################################################
+#
+# con2fbmap system
+#
+#############################################################
+
+
+CON2FBMAP_VERSION = 1.0
+CON2FBMAP_SITE = $(TOPDIR)/../con2fbmap
+CON2FBMAP_SITE_METHOD = local
+CON2FBMAP_LICENSE = BSD-3c
+CON2FBMAP_LICENSE_FILES = LICENSE.txt
```

Comment:
You cherry picked some files from another project, but forget to copy the file that describes the license the files are under.

And fbset looks more like GPL-2.0 rather than BSD-3c

Reply:
Thanks. I'm not very proficient with licenses. I'll update appropriately.

Reply:
Unfortunately it's something you need to be quite careful with, because some people get very upset if you get it wrong.